cfgov/unprocessed/apps/filing-instruction-guide/js/fig-search.js

```diff
@@ -79,23 +79,22 @@ const getSearchData = (sections) => {
  * @param {object} event - Search result follow event
  */
 const onFollow = (event) => {
-  const target = event.target
-    .closest('a')
-    .getAttribute('href')
-    .replace('#', '');
+  const target = event.target.closest('a');
+  const id = target.getAttribute('href').replace('#', '');
+  const label = target.querySelector('h4').innerText;
```

Comment:
MINOR: `id` and `label` are pretty generic. If I were to search the codebase for use of these I would possibly get a lot of noise from HTML `id`s and `<label>`s. Maybe name these the specific things they are? `figLinkID` and `figHeadingLabel` or similar?

Reply:
I dig it. 👍